compiler/rustc_target/src/spec/windows_msvc_base.rs

```diff
@@ -27,6 +27,7 @@ pub fn opts() -> TargetOptions {
         // linking some libraries which require a specific agreement, so it may
         // not ever be possible for us to pass this flag.
         no_default_libraries: false,
+        has_elf_tls: true,
```

Comment:
I think this field wants a rename to something like `has_thread_local` or something of a similar nature?

Reply:
`has_thread_local` sounds ok to me! If anybody wants to bikeshed the name I don't mind changing it again but it's probably not worth arguing about it too much 🙂.

Reply:
Are we sure that this field is never used in some ELF-specific way?

Reply:
Hm, well if it is then it was wrong to use it for msvc targets. But I did do a [search](https://github.com/rust-lang/rust/search?q=has_elf_tls) and the only place it's used (other than specs) is here: https://github.com/rust-lang/rust/blob/c5ecc157043ba413568b09292001a4a74b541a4e/compiler/rustc_session/src/config.rs#L883-L885

As far as I understand it, Rust defers to LLVM for actually generating the TLS access (it uses the LLVM IR `thread_local`).